### PR TITLE
Part of https://github.com/pulibrary/orangelight/issues/4095

### PR DESCRIPTION
**Pull request by christinach**
part of #4095

Remove punctuation dot from the end of a fast subject heading string

### CODE REVIEW

File: marc_to_solr/lib/traject_config.rb [hunk 1]
```diff
@@ -895,7 +895,7 @@
   next if subjects.present?
   record.fields.select { |field| field.tag[0] == '6' }.each do |field|
     next unless field['2'] == 'fast'
-    value = field['a']
+    value = field['a'].delete('.')
```
Comment (sandbergja) [inline marc_to_solr/lib/traject_config.rb:898]: Is there any chance `field['a']` could return nil?
```suggestion
    value = field['a']&.delete('.')
```

**Review by sandbergja — APPROVED** [review]
Thanks, @christinach !  Just one small question

File: marc_to_solr/lib/traject_config.rb (hunk 1 above)
Comment (sandbergja) [reply marc_to_solr/lib/traject_config.rb:898]: Oh, actually one more question sorry!  Wouldn't this remove periods within the subfield a, not just at the end?  I wonder if there are some fast headings that legitimately have "." inside them.

Comment (christinach) [reply marc_to_solr/lib/traject_config.rb:898]: yes it would remove them. I handled the period failure in orangelight so maybe I can close this PR?

Comment (christinach) [reply marc_to_solr/lib/traject_config.rb:898]: @sandbergja I will wait to discuss this with @mzelesky .

Comment (christinach) [reply marc_to_solr/lib/traject_config.rb]: @sandbergja I used delete_suffix to delete the . at the end of the string.

**Review by sandbergja — APPROVED** [review]
Thanks, @christinach 🐬

**Pull request merged**